lib/internal/streams/writable.js

```diff
@@ -216,7 +223,7 @@ if (typeof Symbol === 'function' && SymbolHasInstance) {
   realHasInstance = FunctionPrototype[SymbolHasInstance];
   ObjectDefineProperty(Writable, SymbolHasInstance, {
     value: function(object) {
-      if (realHasInstance.call(this, object))
+      if (ReflectApply(realHasInstance, this, [object]))
```

Comment:
<https://github.com/nodejs/node/pull/36329> will make it possible to use:
```js
FunctionPrototypeSymbolHasInstance(this, object)
```

Also, the above
```js
if (typeof Symbol === 'function' && SymbolHasInstance)
```
check is unnecessary, as it will always return `true`.

Reply:
> Also, the above check is unnecessary, as it will always return `true`.

I think that check is there because https://www.npmjs.com/package/readable-stream is supposed to work with Node.js 6+ and up to Chromium 10, and `Symbol` was introduced in Chromium 38.

Reply:
we'll bump the major for the build from node 14, don't worry about it.